test/vault_helpers.go

```diff
@@ -435,6 +513,31 @@ func initializeAndUnsealVaultCluster(t *testing.T, asgNameOutputVar string, sshU
 	return cluster
 }
 
+func testRequestSecret(t *testing.T, terraformOptions *terraform.Options, expectedResponse string) {
+	instanceIP := terraform.Output(t, terraformOptions, OUTPUT_AUTH_CLIENT_IP)
+
+	curlCommand := shell.Command{
+		Command: "curl",
+		Args:    []string{"-s", fmt.Sprintf("%s:%s", instanceIP, "8080")},
+	}
```

Comment:
No need to do it this way. See https://github.com/gruntwork-io/terratest/blob/master/modules/http-helper/http_helper.go#L91 and other related methods.